libs/features/personalization/personalization.js

```diff
@@ -131,20 +131,45 @@ const CREATE_CMDS = {
   appendtosection: 'beforeend',
 };
 
+const COMMANDS_KEYS = {
+  remove: 'remove',
+  replace: 'replace',
+  update: 'update',
+};
+
+const ATTRIBUTES_TO_UPDATE = {
```

Comment:
Can we rename this updateFlags?  Also think it makes sense to move this inside of the update command declaration in 20 lines to help make the scope of where this is used clear.  The use of all caps and placing it at the top is more for constants to you use in multiple functions.